docs/graphcast/radios/poi-radio.md

```diff
@@ -12,37 +12,89 @@ The key requirement for an Indexer to earn indexing rewards is to submit a valid
 
 ## Getting Started
 
-Before you follow any of the instructions below, please make sure you have a Graphcast Operator address registered for your on-chain Indexer address.
+Before you follow any of the instructions below, please make sure you have a Graphcast ID registered for your on-chain Indexer address.
 
 :::tip
-You can connect a Graphcast Operator address to your Indexer address (with a 1:1 relationship) using our very own [Registry contract](https://goerli.etherscan.io/address/0xBFdA8191D1ec09bB8ADc138DAbf413d00DAfb6c8) (on Goerli). You need to use your Indexer wallet to call the `setGossipOperator` function, providing the address you wish to use as an operator.
+You can connect a Graphcast ID address to your Indexer address (with a 1:1 relationship) using our very own **Registry contract**, we have [one for mainnet](https://etherscan.io/address/0x89f97698d6006f25570cd2e31737d3d22aedcbcf) and [one for testnet](https://goerli.etherscan.io/address/0xe88461436D30CAc2d69b70871EC2cdDC32083aed) (on Goerli). You need to use your Indexer or Operator wallet to call the `setGraphcastIDFor` function, providing the Indexer address, as well as the address you wish to use as a Graphcast ID.
```

Comment:
pls update goerli contract to the latest one here: https://goerli.etherscan.io/address/0x26ebbA649FAa7b56FDB8DE9Ea17aF3504B76BFA0.

I think we could maybe add a bit more on graphcast ID address restrictions: 
- the address must be different from the wallet you are calling from
- the address must not be registered with another address (you can check with the read function `graphcastIDRegistered`